### PR TITLE
Lookup should display the LoadPanel when the data is being loaded for a long time while searching (T837176)

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

**Review by ksercs — DISMISSED** [review]

**Review by alexander-kotov-dx — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js [hunk 1]
```diff
@@ -2930,13 +2936,49 @@ QUnit.test("search should be execute after paste", function(assert) {
...
-    } finally {
-        clock.restore();
-        fx.off = originalFX;
-    }
+    [
+        { loadDelay: 200, indicateLoading: false },
+        { loadDelay: 1000, indicateLoading: true }
+    ].forEach(({ loadDelay, indicateLoading }) => {
+        QUnit.test(`search with loading delay = ${loadDelay} should ${indicateLoading ? "" : "not"} lead to the load panel being displayed`, function(assert) {
+            const clock = sinon.useFakeTimers();
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: Looks like we already do this here:
https://github.com/DevExpress/DevExtreme/pull/10984/files#diff-922b6fcabe4f5ae828645b18174c6277R2919

Can we use `this.clock` instead?

File: js/ui/autocomplete.js [hunk 2]
```diff
@@ -161,8 +161,7 @@ var Autocomplete = DropDownList.inherit({
 
     _listConfig: function() {
         return extend(this.callBase(), {
-            pageLoadMode: "none",
-            indicateLoading: false
+            pageLoadMode: "none"
```
Comment (San4es) [inline js/ui/autocomplete.js:164]: Why are we going to remove this?

**Review by San4es — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js (hunk 1 above)
Comment (DokaRus) [reply testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js]: Thank you, I'll fix it

File: js/ui/autocomplete.js (hunk 2 above)
Comment (DokaRus) [reply js/ui/autocomplete.js:164]: The value of this option is already set to "false" (in the parent component)

**Review by San4es — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**